sympytorch/sympy_module.py

```diff
@@ -105,7 +105,7 @@ def sympy(self, _memodict):
         if issubclass(self._sympy_func, sympy.Float):
             return self._sympy_func(self._value.item())
         elif issubclass(self._sympy_func, sympy.UnevaluatedExpr):
-            return self._sympy_func(self._value.item())
+            return self._sympy_func(self._value.item()).doit()
```

Comment:
I don't think this should be here. Ideally a sympy expression -> SympyModule -> .sympy() round trip should be the identity.

Reply:
What motivates adding this? It looks to me like the fix in this PR should work without it.

Reply:
Without this (or something like it), it still calls the `Min` or `Max` function on the unevaluated expression, triggering the error described above. I've rewritten it to prevent evaluation of these functions in `.sympy()` in the same way as in `hide_floats`.